lessons/pt/basics/enum.md

```diff
@@ -31,9 +31,11 @@ Usando isso, é claro que temos uma grande quantidade de funcionalidades, e isso
 A enumeração é o núcleo da programação funcional e é uma coisa incrivelmente útil.
 Ao aproveitar isso combinado com outras vantagens de Elixir, como a documentação sendo um cidadão de primeira classe como acabamos de ver, também pode ser incrivelmente poderoso para o desenvolvedor.
 
+# Common Functions
```

Comment:
I'm unsure how we could translate this one. Maybe `Funções comumente usadas`? Maybe `Funções` alone would do the job?

```suggestion
# Funções
```